tests/rules/data/pdo-stmt-execute-error.php

```diff
@@ -98,4 +98,27 @@ public function bug336Named(PDO $pdo)
         ');
         $stmt->execute([':akid' => 1]); // everything fine
     }
+
+    public function bug422UnnamedPlaceholdersInComment(PDO $pdo)
+    {
+        $query = '
+            Select * from ada
+            where adakzid = 15 -- customer ???
+        ';
+
+        $stmt = $pdo->prepare($query);
+        $stmt->execute([]);
+    }
+
+    public function bug422NamedPlaceholdersInComment(PDO $pdo)
+    {
+        $stmt = $pdo->prepare('SELECT email, adaid FROM ada -- nice :placeholder bro');
+        $stmt->execute([]);
+    }
+
+    public function bug422NamedAndUnnamedPlaceholdersInComment(PDO $pdo)
```

Comment:
please add a additional est which has parameters and a comment in the query

Reply:
:gift_heart: